torchaudio/csrc/CMakeLists.txt

```diff
@@ -39,6 +62,37 @@ if(BUILD_RNNT)
       rnnt/gpu/compute.cu
       )
   endif()
+
+  if (USE_ROCM)
+    hipify(CUDA_SOURCE_DIR ${PROJECT_SOURCE_DIR}/torchaudio/csrc/rnnt/gpu HIP_SOURCE_DIR ${PROJECT_SOURCE_DIR}/torchaudio/csrc/rnnt/hip)
+    if ( NOT HIP_ADD_LIBRARY_FOUND )
+      list(APPEND CMAKE_MODULE_PATH /opt/rocm/hip/cmake)
+      find_package(HIP REQUIRED)
+    endif()
```

Comment:
This looks like the duplicate from the above change, and better be handled in the root CMakeLists.txt?

Reply:
@pnunna93 Here as well, I believe the hipify logic should stay here, since it's about source code modification (hipification), but the cmake stuff can probably move to root CMakeLists.txt?